packages/placement-ordering/src/placement-ordering.jsx

```diff
@@ -102,16 +139,7 @@ export class PlacementOrdering extends React.Component {
     const { orientation, includeTargets } = config;
     const vertical = orientation === 'vertical';
 
-    const ordering = showingCorrect
-      ? buildState(
-          model.choices,
-          model.correctResponse,
-          model.correctResponse.map(id => ({ id, outcome: 'correct' })),
-          { includeTargets }
-        )
-      : buildState(model.choices, session.value, model.outcomes, {
-          includeTargets
-        });
+    this.ordering = this.createOrdering();
```

Comment:
I'm not comfortable with this, I don't mind using class fields if it's not part of the component's render/state lifecycle, but 'ordering' is directly related to render. If you want you can do: 
```javascript

//... 
onDropChoice(source, target, ordering){ 
   ///...
}
//
render(){
//...
<OrderedTiles ordering={this.createOrdering()}  onDropChoice={this.onDropChoice}/>
}
```